ang/test/civicase/Actions--print-merge.spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jasmine */
+
+(function (_, $) {
+  describe('MoveCopyActivityAction', function () {
+    var PrintMergeCaseAction, CasesMockData;
+
+    beforeEach(module('civicase', 'civicase.data'));
+
+    beforeEach(inject(function (_PrintMergeCaseAction_, _CasesData_) {
+      PrintMergeCaseAction = _PrintMergeCaseAction_;
+      CasesMockData = _CasesData_;
+    }));
+
+    describe('getPath()', function () {
+      var caseObj, returnValue;
+
+      beforeEach(function () {
+        caseObj = CasesMockData.get().values[0];
+
+        returnValue = PrintMergeCaseAction.getPath([caseObj]);
+      });
+
+      it('returns path to open popup for sending printing/merging', function () {
```

Comment:
The statement `returns path to open popup for sending printing/merging` seems a bit confusing. Do you think we should make it `returns path to open popup for sending the activity for printing/merging` ? It's not a big deal and we can keep it as it too. Your call.

Reply:
I think `returns path for opening popup to print/merge document` is more appropriate. what do you think? thanks for the pointing out the issue :)

Reply:
Looks good to me too